config/swagger.yaml

```diff
@@ -2088,6 +2088,36 @@ paths:
           description: Invalid contract
           schema:
             $ref: '#/definitions/Error'
+  /contract/{contract_address}/balance:
+    get:
+      tags:
+        - external
+        - chain
+      operationId: 'GetContractBalance'
+      description: 'Get Contract balance'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
```

Comment:
I guess requirement [at height and at hash](https://github.com/aeternity/epoch/blob/0bdbb40a4da982f6fab39ff9b3c38146e266b35c/config/swagger.yaml#L1288-L1298) will come? At hash at least. (Not sure how to interpret height in Bitcoin-NG.)

Reply:
Probably. Not requested yet